header.php

```diff
@@ -88,8 +88,8 @@
 			</div>
 			<div class="header__nav">
 				<a class="header__nav-icon js-header-nav-toggle" href="#navigation"><?php _e( 'Toggle Menu', 'pressbooks-book' ); ?><span class="header__nav-icon__icon"></span></a>
-				<nav aria-labelledby="book-toc" class="js-header-nav" id="navigation">
-					<p id="book-toc" class="screen-reader-text">Book Contents Navigation</p>
+				<nav aria-labelledby="primary-nav" class="js-header-nav" id="navigation">
+					<p id="primary-nav" class="screen-reader-text">Primary Navigation</p>
```

Comment:
Applied this label in wrong place last time. Whoops!

Reply:
You're right I was confused too